### PR TITLE
feat: update collections layout

### PR DESCRIPTION
**Pull request by karrui**

### CODE REVIEW

**Comment by karrui** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/opengovsg/isomer/462?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#462** <a href="https://app.graphite.dev/github/pr/opengovsg/isomer/462?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#460** <a href="https://app.graphite.dev/github/pr/opengovsg/isomer/460?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#459** <a href="https://app.graphite.dev/github/pr/opengovsg/isomer/459?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @karrui and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/components/src/templates/next/layouts/Collection/CollectionClient.tsx [hunk 1]
```diff
@@ -1,402 +1,153 @@
...
+            <div className="flex h-full w-full flex-col gap-0">
+              {paginatedItems.length > 0 &&
+                paginatedItems.map((item) => (
+                  <CollectionCard
+                    key={Math.random()}
+                    {...item}
+                    LinkComponent={LinkComponent}
+                  />
+                ))}
+              {paginatedItems.length === 0 && searchValue !== "" && (
```
Comment (dcshzj) [inline packages/components/src/templates/next/layouts/Collection/CollectionClient.tsx:114]: issue: I think this condition is too specific. If the user solely filtered using the filters on the left side and `paginatedItems.length` is 0, then the message does not appear.

Screenshot 2024-08-16 at 10 43 19

File: packages/components/src/templates/next/components/internal/Filter/Filter.tsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ const Filter = ({
               <label
                 key={itemId}
                 htmlFor={itemId}
-                className="flex w-full flex-row rounded px-1 py-2 align-middle hover:bg-interaction-main-subtle-hover has-[:focus]:ring-2 has-[:focus]:ring-focus-outline"
+                className="flex w-full flex-row rounded px-1 py-2 align-middle has-[:focus]:ring-2 has-[:focus]:ring-focus-outline hover:bg-interaction-main-subtle-hover"
```
Comment (dcshzj) [inline packages/components/src/templates/next/components/internal/Filter/Filter.tsx:48]: nit: the alignment for the filter checkbox isn't ideal, not middle aligned

Screenshot 2024-08-16 at 10 44 33

Comment (karrui) [reply packages/components/src/templates/next/components/internal/Filter/Filter.tsx:48]: this one is old filter you write one HAHA. Just added format. The new filter in next PR

File: packages/components/src/templates/next/layouts/Collection/CollectionClient.tsx (hunk 1 above)
Comment (karrui) [reply packages/components/src/templates/next/layouts/Collection/CollectionClient.tsx:114]: never touch filter behavior here. This PR solely for layout changes hehe.

I think @sehyunidaaa is working on empty screen styling, will fix alongside that PR.

**Review by dcshzj — APPROVED** [review]

**Comment by karrui** [comment]
### Merge activity

* **Aug 16, 12:39 AM EDT**: @karrui merged this pull request with [Graphite](https://app.graphite.dev/github/pr/opengovsg/isomer/462).

**Pull request merged**